assets/css/app.css

```diff
@@ -1 +1,70 @@
-/* This file is for your main application css. */
\ No newline at end of file
+* {
+  box-sizing: border-box;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif;
+  font-weight: 400;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+.container {
+  align-content: center;
+  align-items: center;
+  display: grid;
+  justify-content: center;
+  justify-items: center;
+}
+
+.text-center {
+  text-align: center;
+}
+
+h1 {
+  font-size: 3rem;
+  font-weight: 500;
+  margin: 2rem;
+}
+
+.magic {
+  background-image: url('/images/caiquenator.jpg');
```

Comment:
I can try to get a higher resolution of this picture if needed 😆

Reply:
Let's check it later :)